src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -615,6 +789,13 @@ void QTClusterFinder::createConsensusFeature_(ConsensusFeature& feature,
   {
     return feature_distance_(left->getFeature(), right->getFeature()).second;
   }
+
+  bool QTClusterFinder::distIsOutlier_(double dist, double rt)
+  {
+    if (bin_tolerances_.empty()) return false;
+    auto it = bin_tolerances_.lower_bound(rt);
```

Comment:
- if rt is slightly larger than last this would return .end() is this still correct?

Reply:
yes, due to the "--" no?